sdk/tests/conformance2/buffers/buffer-type-restrictions.html

```diff
@@ -88,16 +88,6 @@
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, messagePrefix + "WORK");
   else
     wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, messagePrefix + "FAIL");
-
-  if ((firstTarget == gl.TRANSFORM_FEEDBACK_BUFFER && secondTarget != gl.TRANSFORM_FEEDBACK_BUFFER) ||
-      (firstTarget != gl.TRANSFORM_FEEDBACK_BUFFER && secondTarget == gl.TRANSFORM_FEEDBACK_BUFFER)) {
-    bind(firstBindFn, firstTarget, buffer);
-    bind(secondBindFn, secondTarget, buffer);
-
-    var message = "Binding buffer first with " + firstBindFn + " to gl." + firstTargetStr
-    + " and simultaneously binding buffer with " + secondBindFn + " to gl." + secondTargetStr + " should FAIL";
-    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, message);
-  }
```

Comment:
To avoid having the WebGL 2.0.0 conformance tests break on newer browsers, can you make this same change in the conformance-suites/2.0.0/ directory?

Reply:
Done.